frontend/css/styles.css

```diff
@@ -49,3 +49,15 @@
 	max-width: 800px;
 	margin-left: 0; /* Align the container to the left */
 }
+
+/* Game Margin Chart custom size */
+#gameMarginChart {
+    width: 600px !important;
+    height: 600px !important;
+}
```

Comment:
You can use one single selector if you are going to apply the same style to both. Add same class to both